environment_dev.yml

```diff
@@ -0,0 +1,14 @@
+# Environment for developing the conda-store-ui and for running 
+# playwright tests
+
+name: test-env
+channels:
+- conda-forge
+dependencies:
+- python=3.10
+- yarn
+- nodejs==16.14.2
```

Comment:
Why did we pinned to this version?

Reply:
This came from the conda env requirements you sent me. So I guess I'll ask the same question to you :)